dist/data.js

```diff
@@ -10721,7 +10826,8 @@ const THE_OBJ = (function () {
             "of": null,
             "of_ptr": false,
             "type": "method",
-            "params": []
+            "params": [],
+            "description_en": "Returns the current item. This should only be used in an item script, otherwise an invalid Item is returned."
```

Comment:
Item On Die seems to return an invalid Item as well, I think this should be noted